crates/libs/bindgen/src/constants.rs

```diff
@@ -12,7 +12,11 @@ pub fn gen(gen: &Gen, def: Field) -> TokenStream {
 
         if ty == constant_type {
             if ty == Type::String {
-                let crate_name = gen.crate_name();
+                let crate_name: TokenStream = if gen.sys && !gen.standalone {
```

Comment:
This is a bit problematic since this assumes that sys-style code gen should assume a dependency on `windows-core` which is designed for non-sys style code gen. I'd like to avoid adding a "core" crate for `windows-sys` but this conflates the two.

Reply:
Yep, it's not a simple fix but I'd prefer to find a good solution rather than fix a problem by introducing another problem. 😉

Reply:
ok